.github/workflows/swift-toolchain.yml

```diff
@@ -3374,6 +3535,11 @@ jobs:
           name: Android-sdk-${{ matrix.arch }}
           path: ${{ github.workspace }}/BuildRoot/Library/Developer/Platforms/Android.platform
 
+      - uses: actions/download-artifact@v4
+        with:
+          name: ds2-Android-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BuildRoot/Library/Developer/Platforms/Android.platform/Developer/Library/Android/${{ matrix.cpu }}-unknown-linux-android
```

Comment:
I suppose that it doesn't matter too much, but this triple is not entirely correct in all cases - `armv7-unknown-linux-androideabi` is the the triple for ARMv7.

Reply:
It is worth fixing. The packaging manifest expects it in a directory with the proper triple name.